app/models/form.rb

```diff
@@ -5,20 +5,38 @@ class Form
 
   include Form::Setup
 
-  def initialize(form_path, name)
-    raise "No form definition file exists for given year".freeze unless File.exist?(form_path)
-
-    @name = name
-    @setup_sections = [Form::Setup::Sections::Setup.new(nil, nil, self)]
-    @form_definition = JSON.parse(File.open(form_path).read)
-    @form_sections = form_definition["sections"].map { |id, s| Form::Section.new(id, s, self) }
-    @type = form_definition["form_type"]
-    @sections =  setup_sections + form_sections
-    @subsections = sections.flat_map(&:subsections)
-    @pages = subsections.flat_map(&:pages)
-    @questions = pages.flat_map(&:questions)
-    @start_date = Time.iso8601(form_definition["start_date"])
-    @end_date = Time.iso8601(form_definition["end_date"])
+  def initialize(form_path, name, sections_in_form = [], type = "lettings")
+    if type == "sales"
+      # @form_definition = sections.map do |s|
+      #   s.new(nil, nil, self)
+      # end
+      @name = name
```

Comment:
Does a form actually need a name?

Reply:
Probably not, but it doesn't hurt for distinguishing between the 4 forms we'll need to run